autorally_core/launch/stateEstimator.launch

```diff
@@ -1,19 +1,48 @@
+<?xml version="1.0"?>
+
+
+<!-- singlePlatform.launch
+
+Use Gazebo to simulate a vehicle with Ackermann steering.
+
+Arguments:
+    sim (boolean, default: false)
+        parameter used to run state estimator in simulation.
+    InvertZ (boolean, default: true)
+        if the state estimator should invert the Z axis of the imu
+    InvertY (boolean, default: true):
+        if the state estimator should in the Y axis of the imu
+    namespace (string, default: "")
+        the namespace of the topic published by gazebo
+    FixedInitialPose (boolean, default: false)
+        if a fixed initial pose should be used
+-->
+
 <launch>
 
+
+
   <include file="$(find autorally_core)/launch/hardware.machine" />
   <include file="$(find autorally_core)/launch/wheel_odometry.launch" />
+  <arg name="sim" default="false"/>
+  <arg name="namespace" default=""/>
+  <arg name="InvertY" default="true"/>
```

Comment:
Why are these args defined here, then use below?

Reply:
I pass in non default values for these args in the singlePlatform.launch file but is there a better way of doing this?